### PR TITLE
Fix: fixed UI bugs sprint 2

### PR DESCRIPTION
**Pull request by Abhishek9503**
## Description

#1188 

In this Pr i have solved the bug 
1.[Workspaces & Teams] “No result was found” message is missing in list view.
2.Fixed the colour of scrollbar  in Team and worksapce section also in save request section.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [ ] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by pratikshinde590** [comment]
Please attach the issue link

File: src/packages/@workspaces/features/save-as-request/layout/SaveAsRequest.svelte [hunk 1]
```diff
@@ -1007,4 +1007,13 @@
...
   }
+
+
+  ::-webkit-scrollbar-thumb {
+    background-color: var(--bg-tertiary-100);
+  }
+
+  ::-webkit-scrollbar-button {
+    color: var(----bg-tertiary-100);
+  }
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/save-as-request/layout/SaveAsRequest.svelte:1018]: color name ???

File: src/packages/@teams/features/recent-workspace/layout/RecentWorkspace.svelte [hunk 2]
```diff
@@ -59,6 +59,15 @@
 <!-- </section> -->
 
 <style>
+  .sidebar-workspace-list::-webkit-scrollbar-thumb {
+    background-color: var(--bg-secondary-330);
+    /* background-color: red; */
```
Comment (itsmdasifraza) [inline src/packages/@teams/features/recent-workspace/layout/RecentWorkspace.svelte:64]: comment not required

File: src/packages/@teams/features/recent-apis/layout/RecentApi.svelte [hunk 3]
```diff
@@ -42,6 +42,15 @@
 <!-- </section> -->
 
 <style>
+
+.sidebar-recentapi-list::-webkit-scrollbar-thumb {
+    background-color: var(--bg-secondary-330);
+    /* background-color: red; */
```
Comment (itsmdasifraza) [inline src/packages/@teams/features/recent-apis/layout/RecentApi.svelte:48]: comment not required

File: src/packages/@workspaces/features/save-as-request/layout/SaveAsRequest.svelte (hunk 1 above)
Comment (itsmdasifraza) [reply src/packages/@workspaces/features/save-as-request/layout/SaveAsRequest.svelte:1018]: trigger ::webkit with parent

**Review by itsmdasifraza — CHANGES_REQUESTED** [review]

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by itsmdasifraza** [comment]
please use issue hash in commit message ex-
`feat: commit message [#1095]`

File: src/packages/@app/styles/style.scss [hunk 4]
```diff
@@ -318,6 +319,7 @@ $theme-red-230408: #230408;
   --bg-secondary-290 : #{$theme-grey-484848};
   --bg-secondary-300 : #{$theme-grey-45494D};
   --bg-secondary-320 : #{$theme-grey-353646};
+  --bg-secondary-330 : #{$theme-grey-333333};
```
Comment (Astitva877) [inline src/packages/@app/styles/style.scss:322]: do also add these for icon, border, text

**Review by Astitva877 — APPROVED** [review]

**Review by itsmdasifraza — APPROVED** [review]

**Pull request merged**